### DISCUSSION TITLE
Show and Tell: Interface 2-D coupled Poisson's problem

### DISCUSSION THREAD

**Discussion by AnthonyB08** [dpploy/engy-5310#66] [Show and tell]
Hi all,

Although, I will most likely not be presenting on my future work. I thought I share my 2-D interface coupled poisson problem results!

![image](https://user-images.githubusercontent.com/55299169/117542173-8cd3e280-afe5-11eb-808b-9590e8214cb1.png)

![image](https://user-images.githubusercontent.com/55299169/117542191-a543fd00-afe5-11eb-91b9-ceec93f0e3d7.png)

![image](https://user-images.githubusercontent.com/55299169/117542222-c1479e80-afe5-11eb-8a1a-9afa2fd2f378.png)

![image](https://user-images.githubusercontent.com/55299169/117542253-df150380-afe5-11eb-8685-0621719bcccc.png)

**Comment by dealmeidavf** [comment]
Thanks for showing. You are making progress.
